docs/developers/meta/sc-config.md

```diff
@@ -82,7 +88,38 @@ The settings are as follows:
     - _default_: `[]`
 - `kill-legacy-callback`
     - The framework has no way of knowing whether or not a certain smart contract variant actually needs the async callback code or not, so in rare situations it is necessary to forcefully remove it.
-    - _values_: `true` | `false
+    - _values_: `true` | `false`
+    - _default_: `false`
+- `codegen-units`
+    -  Controls the number of "code generation units" a crate will be split into. Splitting a crate into multiple code generation units can have a significant impact on the compile time and code optimization of the crate.
+    - _default_: `1`
+- `opt-level`
+    - Controls the level of optimization that Rust will apply to the code during compilation.
+    - _values_:
```

Comment:
Add:
- By default we run with `s` or `z`, since for smart contracts bytecode size optimization is of the essence.
- We also run `wasm-opt` _after_ this optimization phase, so this only refers to part of the optimization.